test/e2e/core.go

```diff
@@ -62,7 +63,10 @@ func CoreDump(dir string) {
 		cmds = append(cmds, []command{
 			{"cat /var/log/kubelet.log", "kubelet"},
 			{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+			{"cat /var/log/supervisor/kubelet-stdout.log", "supervisord-kubelet-stdout"},
+			{"cat /var/log/supervisor/kubelet-stderr.log", "supervisord-kubelet-stderr"},
 			{"cat /var/log/kern.log", "kern.log"},
+			{"cat /var/log/docker.log", "docker.log"},
```

Comment:
don't need the `.log` in the second string - it gets added automatically. i.e. this should be
```go
{"cat /var/log/docker.log", "docker"},
```

(similar for other uses. `kern.log` is likewise incorrect, though I missed that earlier)